Doc/whatsnew/3.12.rst

```diff
@@ -486,6 +491,18 @@ Porting to Python 3.12
   :c:func:`PyUnicode_FromFormatV`.
   (Contributed by Philip Georgi in :gh:`95504`.)
 
+* Extension classes wanting to add a ``__dict__`` or weak reference slot
+  should use :const:`Py_TPFLAGS_MANAGED_DICT` and
+  :const:`Py_TPFLAGS_MANAGED_WEAKREF` instead of ``tp_dictoffset`` and
+  ``tp_weaklistoffset``, respectively.
+  The use of ``tp_dictoffset`` and ``tp_weaklistoffset`` is still
+  supported, but will not fully support multiple inheritance
```

Comment:
```suggestion
  supported, but does not fully support multiple inheritance
```

I was a little surprised by the tense here, but I also may be wrong.

Reply:
You are right.
Because this doesn't apply right now, I unthinkingly use the future tense.